client/src/partials/projects/projects.html

```diff
@@ -44,120 +40,188 @@
   </div>
 </div>
 
-<main class="hidden-print">
-  <div class="row">
-
-    <div class="col-xs-6">
-      <div class="panel panel-primary">
-        <div class="panel-heading">
-          {{ 'PROJECT.EXISTING' | translate }}
+<main class="flex-content">
```

Comment:
To be consistent with @sfount's new style, this element should be a `<div>` not a `<main>`.  The main element may still inherit old bhima-1.X styles.